hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -123,6 +127,18 @@ public void setBlockDeletionLimit(int limit) {
   private long periodicDiskCheckIntervalMinutes =
       PERIODIC_DISK_CHECK_INTERVAL_MINUTES_DEFAULT;
 
+  @Config(key = "failed.volumes.tolerated",
+      defaultValue = "0",
```

Comment:
I feel, we should make the default as -1 rather than zero by default.

Reply:
@bshashikant here I borrowed the default 0 from HDFS to stay aligned, but I also think that -1 is reasonable.
With -1 as the default, we introduce no behavioral change to existing clusters when we have a few(1~2) volumes failed.
Will update, thanks.